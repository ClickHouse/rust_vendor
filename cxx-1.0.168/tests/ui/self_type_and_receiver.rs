#[cxx::bridge]
mod ffi {
    unsafe extern "C++" {
        type T;

        #[Self = "T"]
        fn method(self: &T);
    }
}

fn main() {}
