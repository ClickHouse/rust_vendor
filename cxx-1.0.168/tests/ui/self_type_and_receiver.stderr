error: function with Self type must not have a `self` argument
 --> tests/ui/self_type_and_receiver.rs:6:18
  |
6 |         #[Self = "T"]
  |                  ^^^
